### DISCUSSION TITLE
Base table or view not found

### DISCUSSION THREAD

**Discussion by kofesto** [laravel/framework#39334] [Q&A]
I have the following errors when creating migration 
 A table was not found: You might have forgotten to run your migrations. You can run your migrations using `php artisan migrate`.
    https://laravel.com/docs/master/migrations#running-migrations

  1   C:\xampp\htdocs\salesplat\vendor\laravel\framework\src\Illuminate\Database\Connection.php:485
      PDOException::("SQLSTATE[42S02]: Base table or view not found: 1146 Table 'salesplat.user_verifications' doesn't exist")

  2   C:\xampp\htdocs\salesplat\vendor\laravel\framework\src\Illuminate\Database\Connection.php:485
      PDOStatement::execute()

**Comment by kofesto** [comment]
`<?php

use Illuminate\Database\Migrations\Migration;
use Illuminate\Database\Schema\Blueprint;
use Illuminate\Support\Facades\Schema;

class CreateUserVerificationsTable extends Migration
{
    /**
     * Run the migrations.
     *
     * @return void
     */
    public function up()
    {
        Schema::table('user_verifications', function (Blueprint $table) {
            $table->id();
            $table->unsignedInteger('user_id');
            $table->string('token');
            //$table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
            //$table->timestamps();
        });
    }

    /**
     * Reverse the migrations.
     *
     * @return void
     */
    public function down()
    {
        Schema::dropIfExists('user_verifications');
    }
}
`

**Comment by hmennen90** [comment]
Try to replace Schema::table with Schema::create

Schema::table is for altering existing Tables

Schema::create creates a new Table

**Reply by kofesto** [reply]
Thanks it works now

**Reply by kofesto** [reply]
> Thanks it works now
